BlogTemplate/wwwroot/css/site.css

```diff
@@ -47,6 +47,10 @@ nav {
             margin: 0;
         }
 
+        nav li:nth-child(5){

```

Comment:
The 5 feels fragile to me.  If another item gets added into the list before that, it'll throw off your intent.